.env.dev.sample

```diff
@@ -4,3 +4,4 @@ DB_PASSWORD=postgres
 DB_HOST=localhost
 DB_PORT=5432
 HOST_URL=http://localhost:4000
+SECRET_KEY=<insert secret key>
```

Comment:
```suggestion
GUARDIAN_SECRET_KEY=<insert secret key>
```

Would be better :pray:

Reply:
I think that adding a comment on how to generate it would be nice too.